inc/routing_table.h

```diff
@@ -11,7 +11,7 @@
 /*******************************************************************************
  * Definitions
  ******************************************************************************/
-#define MAX_RTB_ENTRY 40
+#define MAX_RTB_ENTRY 60
```

Comment:
We should keep 40 but put it at default value with a `#ifdef` allowing a reconfiguration on product_config.h